erica_app/erica/request_processing/erica_input.py

```diff
@@ -89,107 +101,95 @@ class FormDataEst(BaseModel):
     def if_no_tax_number_must_be_submission_without_tax_nr(cls, v, values):
         if values.get('steuernummer') and v:
-            raise ValidationError('can not be a new admission if tax number given')
+            raise ValueError('can not be a new admission if tax number given')
         if not v and not values.get('steuernummer'):
-            raise ValidationError('must be new admission if no tax number given')
+            raise ValueError('must be new admission if no tax number given')
         return v
 
     @validator('steuernummer')
     def must_be_correct_length(cls, v):
         if v and not 10 <= len(v) <= 11:
-            raise ValidationError('must be 10 or 11 numbers long')
+            raise ValueError('must be 10 or 11 numbers long')
         return v
 
     @validator('bufa_nr', always=True)
     def if_submission_without_tax_nr_bufa_nr_must_be_set_correctly(cls, v, values):
         if values.get('submission_without_tax_nr') and (not v or not len(v) == 4):
-            raise ValidationError('must be 4 numbers long for new admission')
+            raise ValueError('must be 4 numbers long for new admission')
         if values.get('submission_without_tax_nr') and not is_valid_bufa(v):
             raise InvalidBufaNumberError
         return v
 
     @validator('familienstand_married_lived_separated_since', always=True)
     def must_be_set_if_familienstand_married_lived_separated_set(cls, v, values, **kwargs):
         if values.get('familienstand_married_lived_separated') and not v:
-            raise ValidationError('must be set if familienstand_married_lived_separated set')
+            raise ValueError('must be set if familienstand_married_lived_separated set')
         return v
 
     @validator('familienstand_widowed_lived_separated_since', always=True)
     def must_be_set_if_familienstand_widowed_lived_separated_set(cls, v, values, **kwargs):
         if values.get('familienstand_widowed_lived_separated') and not v:
-            raise ValidationError('must be set if familienstand_widowed_lived_separated set')
+            raise ValueError('must be set if familienstand_widowed_lived_separated set')
         return v
 
-
+    @validator('person_a_requests_pauschbetrag')
+    def pauschbetrag_for_person_a_should_not_be_requested_if_no_disability_information_given(cls, v, values):
+        disability_information = [
+            values.get('person_a_disability_degree'),
+            values.get('person_a_has_pflegegrad'),
+            values.get('person_a_has_merkzeichen_bl'),
+            values.get('person_a_has_merkzeichen_tbl'),
+            values.get('person_a_has_merkzeichen_h'),
+            values.get('person_a_has_merkzeichen_g'),
+            values.get('person_a_has_merkzeichen_ag'),
+        ]
+        if not any(disability_information) and v is True:
+            raise ValueError('Person A can not request Pauschbetrag if no disability information given')
```

Comment:
I just notived because you changed it to ValueError for the year: I get it for the year, but these should be ValidationErrors in my head, don't you think? Or is ther a specific reason you decided for ValueError?

Reply:
I noticed that using a ValidationError here resulted in problems because the pydantic validationError.__init__ does expect 3 parameters (self, the attribute where the error occurred and the error message). As far as I understand it, raising a ValueError here will result in a ValidationError when using `.parse_obj`.

That is very different from WTForms validation where the validators should never raise a ValuError but always raise a ValidationError.

That said. I will have another look.

Reply:
I had another look at the [documentation](https://pydantic-docs.helpmanual.io/usage/validators/) and they use ValueError, too. Furthermore, I tried the ValidationError and it works exactly as described above.